src/util/colorScale.ts

```diff
@@ -583,9 +579,36 @@ function parseUserProvidedLegendData(
       });
     if (boundArrays.length===legendValues.length) {
       legendValues.forEach((v, i) => {legendBounds[v]=boundArrays[i];});
+      userProvidedLegendBounds = true;
     } else {
       legendBounds = createLegendBounds(legendValues);
+      userProvidedLegendBounds = false;
```

Comment:
Noticed this in an expected boundary display in a [testing dataset](https://auspice-james-display-n-mpd108.herokuapp.com/flu/seasonal/h1n1pdm/ha/2y?c=num_date) that does not define bounds

<img width="343" height="189" alt="Screenshot 2025-09-29 at 1 37 51 PM" src="https://github.com/user-attachments/assets/9300f4dc-b252-4423-aab0-b7fe27540c3e" />

I'm not sure this is catching _all_ cases where the user does not provide legend bounds? At beginning of the function, it immediately returns `false` if the provided legend is not an array

https://github.com/nextstrain/auspice/blob/26d2f6091376f44775f4a0976cb923ad4a5e1ff2/src/util/colorScale.ts#L547

Reply:
Thanks for the catch! It wasn't the early return it was [a typo elsewhere](https://github.com/nextstrain/auspice/compare/26d2f6091376f44775f4a0976cb923ad4a5e1ff2..3da26eb7221287462dcf7d764a488318e67d68a3). Now looks like this:

<img width="260" height="250" alt="image" src="https://github.com/user-attachments/assets/d712552c-4007-49fa-ad9c-35c26713adf3" />

Reply:
Ah, nice! I'm was confused by this bit in the commit message

> I considered displaying the bounds for _all_ continuous scales but when
the bounds are chosen by Auspice it's less ideal. (Still, maybe we
should show them in a pop-up or similar.)

Now I'm seeing this is referring to the legend _label_ and not the hover text 🤦‍♀️